lib/utils/import/evernote/test.js

```diff
@@ -0,0 +1,30 @@
+import EvernoteImporter from './';
+
+describe('EvernoteImporter', () => {
+  describe('getConvertedDate', () => {
+    let dateNowSpy;
+    let importer;
+
+    beforeAll(() => {
+      dateNowSpy = jest.spyOn(Date, 'now').mockImplementation(() => 1542312140788);
+    });
+
+    beforeEach(() => {
+      importer = new EvernoteImporter({ noteBucket: {}, tagBucket: {} });
+    });
+
+    afterAll(() => {
+      dateNowSpy.mockRestore();
+    });
+
+    it('should return a Unix timestamp, given an un-delimited ISO string', () => {
+      const convertedDate = importer.getConvertedDate('20181008T172440Z');
+      expect(convertedDate).toBe(1539019480);
+    });
+
+    it('should fall back to the current Unix timestamp', () => {
+      const fallbackDate = importer.getConvertedDate('broken-timestamp');
+      expect(fallbackDate).toBe(1542312140.788);
+    });
+  });
+});
```

Comment:
Was this intended for this PR? 🤔

Reply:
Yeah, sorry this PR is a bit of a mess 😅 But the [date parsing interface changed](https://github.com/Automattic/simplenote-electron/pull/1014/commits/333d1c9da3f80662be2be24efa0844ed37a48946#diff-3fbf157f35b018392cc4e6561eced865) as a result of replacing `moment.js` with `date-fns`, and I needed to make sure that it was doing as I intended.

Reply:
Ahh, I see. Thanks for the clarification.